Unit.cpp

```diff
@@ -0,0 +1,28 @@
+#include "Unit.h"
+
+const int Unit::getHp()
+{
+	return hp;
+}
+
+const int Unit::getDmg()
+{
+	return dmg;
+}
+
+const std::string Unit::getName()
+{
+	return name;
+}
+
+void Unit::takeDamage(const int & damage)

```

Comment:
tovabbra is tul altalanos. Igy is akarmennyivel tudjuk valtoztatni. Mi azt szeretnenk, hogy csak egy masik unit amikor meguti, akkor annak adamage-evel csokkenjen, de ne irhassak be akarhova olyat, hogy na, 1231341524-el csokkentse a hp-t csak ugy.

Reply:
Igen, így teljesen világossá vált, hogy mi az elvárás. Az új commitolt verzióban már csak a másik unitra vonatkozó damage értéket vesszük alapul, ezt javítottuk.